crates/optimism/consensus/src/lib.rs

```diff
@@ -155,42 +158,32 @@ impl HeaderValidator for OpBeaconConsensus {
         _total_difficulty: U256,
     ) -> Result<(), ConsensusError> {
         // with OP-stack Bedrock activation number determines when TTD (eth Merge) has been reached.
-        let is_post_merge = self.chain_spec.is_bedrock_active_at_block(header.number);
+        debug_assert!(
+            self.chain_spec.is_bedrock_active_at_block(header.number),
+            "manually import OVM blocks"
+        );
```

Comment:
hmm, I see, technically we don't need this since we expect that bedrock is always activated and we don't need to perform any pre bedrock consensus checks, so this seems fine